deepsourcelabs/zeal-next
# feat:  add type generation

## Description

Add automatic type generation to build command.

### Related tickets, documents

### Type of change




### Changes in this PR

- Set `strict-peer-dependencies` to `false`.
- Add `.prettierignore`.
- Remove extra files.
- Add type generation to build command.
- Update the lock file.

## QA instructions, Previews

1. Clone repo and checkout to branch.
2. Run `pnpm run build`.
3. The `dist` folder should have a `main.d.ts` file.

## What gif best describes this PR or how it makes you feel?

![Types generating all around me](https://media.giphy.com/media/v7WM6sLcnGIc8/giphy.gif)
